### PR TITLE
Bug dfc 4108 migration search autosuggest returns counsellor twice

### PR DESCRIPTION
**Pull request by dineshtripathi**
To remove the duplicate items appearing in the list , the collection should have only distinct item. So the fix is added the distinct item in the collection.

### CODE REVIEW

File: DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs [hunk 1]
```diff
@@ -514,7 +514,7 @@ public void SuggestionsTest(string searchTerm, bool useFuzzyAutoCompleteMatching
                     UrlName = "dummy-url"
                 };
 
-                dummySuggestResult.Results = A.CollectionOfDummy<SuggestionResultItem<JobProfileIndex>>(2);
+                dummySuggestResult.Results = A.CollectionOfDummy<SuggestionResultItem<JobProfileIndex>>(2).GroupBy(x => x.MatchedSuggestion).Select(x => x.First());
```
Comment (trevorkapswarah) [inline DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: Please confirm this how we are grouping on the controller, are we not grouping by label?

Comment (dineshtripathi) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: We are grouping by label as label is holding the matched suggestion

Comment (dineshtripathi) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: To get the distinct value in the dropdown list

Comment (trevorkapswarah) [reply DFC.Digital/DFC.Digital.Web.Sitefinity.JobProfileModule.UnitTests/Controllers/JobProfileSearchBoxControllerTests.cs]: .GroupBy(x => x.MatchedSuggestion) vs ).GroupBy(x => x.label)

**Pull request merged**